src/error-table.rkt

```diff
@@ -166,17 +167,65 @@
         #|
         TODO Make this actually work
         |#
-        [(list (or 'sin.f64 'sin.f32) arg)
+        #;[(list (or 'sin.f64 'sin.f32) arg)
```

Comment:
If you've replaced the original, can you delete this one just to not be confusing?